src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
+    if [[ "${OS}" = "Darwin" ]]; then
+        local mac_version=$(sw_vers -productVersion)
+        if [[ $(awk 'BEGIN {print ("'${mac_version}'" < "'10.16'")}') )) ]]; then
```

Comment:
`awk` is not too dependable across different OS, that's why we've refrained from using it. 
 Even though this will be used only in MacOS, I'd like to suggest something more readable: 

```
       local mac_version=$(sw_vers -productVersion)
       local mac_version_major=$(echo ${mac_version} | cut -f 1 -d '.')
       local mac_version_minor=$(echo ${mac_version} | cut -f 2 -d '.')
       if [[ ${mac_version_major} -lt 10 || ( ${mac_version_major} -eq 10 && ${mac_version_minor} -lt 13 ) ]];
```

Reply:
Good call -- done